crypto3/libs/blueprint/include/nil/blueprint/components/algebra/curves/detail/r1cs/fixed_base_mul_zcash.hpp

```diff
@@ -224,7 +224,7 @@ namespace nil {
                             detail::blueprint_variable<field_type> pad_i;
                             pad_i.allocate(bp);
                             bp.val(pad_i) = field_value_type::zero();
-                            padded_input.template emplace_back(pad_i);
+                            padded_input.template emplace_back<>(pad_i);
```

Comment:
Did you change some flags of compilation? why need to modify code?

Reply:
I have upgraded clang from 18 to 19, compilation fails on this

Reply:
why do you need upgrade clang?